### PR TITLE
[CSL-1336] Ensure wallet nodes work well with `--no-ntp`

### PR DESCRIPTION
**Pull request by pva701**
This PR doesn't solve all our problems w/ `--no-ntp`, it updates genesises and simple slotting.
* We won't use OUR PERFECT NTP in mainnet, hence I added `--no-ntp` flag to dev scripts
* I and @georgeee decided to add field to simple slotting with last returned slot, to avoid situation when we return descending sequence of slots from slotting-functions

### CODE REVIEW

**Comment by pva701** [comment]
Goooosha, approve?!

File: test/Test/Pos/Block/Logic/Mode.hs [hunk 1]
```diff
@@ -298,9 +301,9 @@ instance MonadSlotsData (TestInitMode ssc) where
     putSlottingData = putSlottingDataDefault
 
 instance MonadSlots (TestInitMode ssc) where
-    getCurrentSlot = getCurrentSlotSimple
-    getCurrentSlotBlocking = getCurrentSlotBlockingSimple
-    getCurrentSlotInaccurate = getCurrentSlotInaccurateSimple
+    getCurrentSlot = getCurrentSlotSimple =<< mkSimpleSlottingVar
+    getCurrentSlotBlocking = getCurrentSlotBlockingSimple =<< mkSimpleSlottingVar
+    getCurrentSlotInaccurate = getCurrentSlotInaccurateSimple =<< mkSimpleSlottingVar
     currentTimeSlotting = currentTimeSlottingSimple
```
Comment (georgeee) [inline test/Test/Pos/Block/Logic/Mode.hs]: Is it ok that `currentTimeSlotting` is potentially not coherent with `getCurrentSlotXX`?

Comment (pva701) [inline test/Test/Pos/Block/Logic/Mode.hs]: dunno, I thought it's not a problem.
cc @gromakovsky

File: infra/Pos/Slotting/Impl/Simple.hs [hunk 2]
```diff
@@ -24,30 +28,50 @@ import           Pos.Slotting.Types          (SlottingData (..))
...
+----------------------------------------------------------------------------
+
+data SimpleSlottingState = SimpleSlottingState
+    { _sssLastSlot :: !SlotId
+    }
+
+type SimpleSlottingVar = TVar SimpleSlottingState
+
+mkSimpleSlottingVar :: MonadIO m => m SimpleSlottingVar
+mkSimpleSlottingVar = atomically $ newTVar $ SimpleSlottingState $ unflattenSlotId 0
```
Comment (gromakovsky) [inline infra/Pos/Slotting/Impl/Simple.hs]: There is `newTVarIO` IIRC.

File: test/Test/Pos/Block/Logic/Mode.hs (hunk 1 above)
Comment (gromakovsky) [inline test/Test/Pos/Block/Logic/Mode.hs]: I think we can't do better.

File: test/Test/Pos/Block/Logic/Mode.hs [hunk 3]
```diff
@@ -298,9 +301,9 @@ instance MonadSlotsData (TestInitMode ssc) where
     putSlottingData = putSlottingDataDefault
 
 instance MonadSlots (TestInitMode ssc) where
-    getCurrentSlot = getCurrentSlotSimple
-    getCurrentSlotBlocking = getCurrentSlotBlockingSimple
-    getCurrentSlotInaccurate = getCurrentSlotInaccurateSimple
+    getCurrentSlot = getCurrentSlotSimple =<< mkSimpleSlottingVar
```
Comment (gromakovsky) [inline test/Test/Pos/Block/Logic/Mode.hs]: It doesn't make sense to create a new slotting var every time.

Comment (gromakovsky) [inline test/Test/Pos/Block/Logic/Mode.hs]: On the other hand, in init mode it's probably fine, so I don't mind.

**Pull request merged**